### DISCUSSION TITLE
Map v2.0.17

### DISCUSSION THREAD

**Discussion by furuame** [sparcians/map#519] [Ideas]
I’d like to release the new Map v2.0.17 with my recent Register enhancement: https://github.com/sparcians/map/pull/515

Not sure what the release process is, I’d better ask first before pushing the new tag!

**Comment by knute-mips** [comment]
- Merged changes into master
- Merge changes from master into map_v2 branch
- Tag map_v2 branch with the next tag
- [Make a release](https://github.com/sparcians/map/releases)

**Reply by furuame** [reply]
Thanks! I’ll release v2.0.17 today.

**Comment by knute-mips** [comment]
You should be able to.  You're on the "short list" of folks that can.  😉

**Reply by furuame** [reply]
Yay! 😁
